chrome_agent/manifest.json

```diff
@@ -11,17 +11,16 @@
 
   "background": {
     "persistent": true,
-    "scripts": [
-      "scripts/agent_background.js"
-    ]
+    "page": "background.html"
```

Comment:
`"page"` is unofficially deprecated, and having background.html looks weird for an extension anyway (even for an app). Why not just list the scripts imported in background.html here?